spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher_spec.rb

```diff
@@ -0,0 +1,24 @@
+describe ManageIQ::Providers::Redhat::InfraManager::Refresh::Refresher do
+  let(:ems) { FactoryGirl.create(:ems_redhat) }
```

Comment:
maybe you dont even have to create the ems, but just build it. Less hits on the DB
`FactoryGirl.build(:ems_redhat)`